frontend/ui/bidi.lua

```diff
@@ -283,4 +284,18 @@ function Bidi._filepath_rtl(path)
     return Bidi.ltr(Bidi._path(dirpath) .. Bidi._filename_auto_ext_right(filename))
 end
 
+function Bidi.invert()
+    if G_reader_settings:isTrue("invert_ui_layout_mirroring") then
```

Comment:
No such setting set or used anywhere. So, couldn't trigger your stuff unless I replace this with `if true then`.

Reply:
I did not build out the UI for this setting yet. I was waiting to see if this aproach was viable. the plan is that if **both** `invert_ui_layout_mirroring` and `inverse_reading_order`  are true then this will trigger.